### PR TITLE
Mf history

### PR DESCRIPTION
**Pull request by RVRhub**

### CODE REVIEW

File: history/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# NGINX base image
+FROM nginx:1.18.0-alpine
```
Comment (leafchild0) [inline history/Dockerfile:2]: Why nginx is needed? To just serve static files without a server?

File: history/src/App.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<template>
+	<v-app>
```
Comment (leafchild0) [inline history/src/App.vue:2]: Formatting

File: history/src/components/History.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+<script>
+export default {
+	name: 'history',
+	props: {
+		historyRecords: Array
+	},
+	data () {
+		return {
+			search: '',
```
Comment (leafchild0) [inline history/src/components/History.vue:30]: Formatting

File: history/package.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "history",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "server": "node server.js",
```
Comment (leafchild0) [inline history/package.json:7]: Where is server.js? What is it doing?

**Review by leafchild0 — CHANGES_REQUESTED** [review]
Not clear how this works. Where is server? How it serves data?
Formatting again

File: history/Dockerfile (hunk 1 above)
Comment (RVRhub) [reply history/Dockerfile:2]: This is a server where vueApp is deployed, this is similar to tomcat and will integrate through gateway and so on. I think this is a super simple frontend app

Comment (leafchild0) [reply history/Dockerfile:2]: OK, where to get data? I.e. for displaying the history? Kafka?

Comment (RVRhub) [reply history/Dockerfile:2]: Current Service -> Gateway -> History Data Service

**Pull request merged**